models/tokens/tokens_schema.yml

```diff
@@ -33,4 +33,25 @@ models:
       - name: symbol
         description: "NFT Token's symbol"
       - name: decimals
-        description: "NFT Token's number of decimals"
\ No newline at end of file
+        description: "NFT Token's number of decimals"
+
+  - name: tokens_native
+    meta:
+      blockchain: avalanche_c, bnb, ethereum, gnosis, optimism, ethereum, arbitrum, solana
+      sector: tokens
+      contributors: ilemi
+    config:
+      tags: ['tokens','native', 'avalanche_c', 'bnb','ethereum', 'gnosis','optimism','ethereum','solana','arbitrum']
+    description: >
+        The chain native token and symbol, and the prices.usd symbol/address that is closest.
+    columns:
+      - name: chain
+        description: "the chain, as it's named in Dune" 
+      - name: symbol
+        description: "Token's symbol"
+      - name: price_symbol
+        description: "Token's prices.usd symbol"
+      - name: price_address
```

Comment:
would it make sense to put a unique check on the address? or maybe unique combo of columns that include address and chain?

Reply:
It should always be only one row for chain, so unique on chain should be good enough. Maybe future chains will adopt the "deploy thing to same address everywhere" practice so not really necessary rn